lib/ibm_power_hmc/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module IbmPowerHmc
-  VERSION = "0.8.6"
+  VERSION = "0.8.7"
```

Comment:
Not a big deal but usually would prefer to update the gem version as a separate commit (it's OK to even push that commit directly to _master_ without a PR).